integration_tests/conftest.py

```diff
@@ -134,23 +100,21 @@ def dea_index_instance():
     return dea_index_instance
 
 
-module_index = index_fixture("module_db", scope="module")
-
-module_dea_index = dea_index_fixture("module_index", scope="module")
+module_dea_index_fixture = dea_index_fixture_factory("module_dea_index", scope="module")
```

Comment:
Is this fixture actually being used anywhere? Either way, the `dataset_loader` fixture needs to be updated accordingly